src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -775,6 +775,23 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 					}
 					this.openerService.open(file, options);
 				}));
+
+				store.add(dom.addDisposableListener(widget, dom.EventType.KEY_DOWN, (e: KeyboardEvent) => {
```

Comment:
Key press instead?

Reply:
i think `KEY_DOWN` is correct, `KEY_PRESS` isn't used anywhere else in core atm it seems